app/index.ejs

```diff
@@ -4,13 +4,22 @@
 <head>
   <meta name=viewport content="width=device-width, user-scalable=no">
   <meta charset="utf-8">
+  <meta name="msapplication-TileColor" content="#603cba">
+  <meta name="theme-color" content="#ffffff">
   <title>Cosmos Voyager</title>
+
+  <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="mask-icon" href="safari-pinned-tab.svg" color="#5bbad5">
+
   <% if (htmlWebpackPlugin.options.appModules) { %>
-    <% } %>
+  <% } %>
 
-      <body style="background: #15182d">
-        <script async src='https://www.google-analytics.com/analytics.js'></script>
-        <div id="app" style="background: #15182d"></div>
-      </body>
+<body style="background: #15182d">
+  <script async src='https://www.google-analytics.com/analytics.js'></script>
```

Comment:
unrelated to this PR: we should probably bundle this instead of making this request here? what do you guys think?

@sabau @faboweb @fedekunze

Reply:
I think it's ok to keep it in the main template, those resources won't change often

Thanks for fixing the indentation here 👍